pgsqlite.py

```
@@ -0,0 +1,1026 @@
+                                        END
+                                    )
+                                """).format(
+                                    table=Identifier(table_name),
+                                    col=Identifier(transpiled_col),
+                                )
+                            )
+                            logger.info(
+                                f"Converted {table_name}.{transpiled_col} "
+                                f"from {current_type} to jsonb"
+                            )
+                except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)